packages/account-v2/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "@deriv/account-v2",
+    "version": "2.0.0",
+    "main": "dist/account-v2/js/index.js",
+    "engines": {
+        "node": "18.x"
+    },
+    "scripts": {
+        "build": "rimraf dist && NODE_OPTIONS='-r ts-node/register' webpack --progress --config \"./webpack.config.js\" && npm run translate",
+        "serve": "rimraf dist && concurrently \"cross-env BUILD_MODE='serve' NODE_OPTIONS='-r ts-node/register' webpack --progress --watch --config ./webpack.config.js\" \"tsc -w --noEmit --preserveWatchOutput\"",
+        "start": "rimraf dist && npm run test && npm run serve",
+        "translate": "sh ./scripts/update-translations.sh"
+    },
+    "dependencies": {
+        "@deriv/quill-design": "^1.2.12",
+        "classnames": "^2.3.2",
+        "i18n-iso-countries": "^7.7.0",
+        "i18next": "^23.7.11",
+        "moment": "^2.29.4",
+        "qrcode.react": "^3.1.0",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
+        "react-dropzone": "11.0.1",
+        "react-i18next": "^11.11.0",
+        "react-router-dom": "^5.2.0",
```

Comment:
let's discover if there's a possibility to upgrade to v6

Reply:
Will check with @ako.
But I think there will be error due to mismatch of version between the main package json and the packagejson of accounts-v2. I faced this issue with libs like classnames, stylelint